src/scripts/verify-revision-is-deployed.sh

```diff
@@ -0,0 +1,55 @@
+# These variables are evaluated so the config file may contain and pass in environment variables to the parameters.
+ECS_PARAM_FAMILY=$(eval echo "$ECS_PARAM_FAMILY")
```

Comment:
In the event the users passes a string like "$MYVALUE" we would need to eval this to get the true value they intended. That was applied to anything that I felt might be "secretive" or possibly something they may store twice. Because this is just a number, I figured it was unlikely the user would enter something like `$MAX_ATTEMPTS`, though we could add it.